bin/studio-common

```diff
@@ -64,14 +64,18 @@ fi;
 
 if $need_upgrade; then
   echo "ERROR: Habitat needs to be upgraded to version '$SUPPORTED_HAB_VERSION'";
-  echo "       Exit the studio and run the command:";
+  echo "       Run:";
+  echo "       => hab hab studio rm";
```

Comment:
should this be `hab studio rm` or `hab hab studio rm`?

Reply:
Ha ha, `hab studio rm`. Thanks for catching my typo.

Reply:
This is fixed now